GDJS/Runtime/howler-sound-manager/howler-sound-manager.js

```diff
@@ -485,17 +528,42 @@ gdjs.HowlerSoundManager.prototype.unloadAudio = function (soundName, isMusic) {
 
   if (!container[soundFile]) return;
 
+  // Make sure any sound using the howl is deleted so that the howl can be garbage collected
+  // and no weird "zombies" using the unloaded howl can exist.
+  const howl = container[soundFile];
+  function removeFromConatiner(soundContainer) {
+    for (let i in soundContainer) {
+      if (soundContainer[i] && soundContainer[i]._howl === howl) {
+        soundContainer[i].stop();
+        delete soundContainer[i];
+      }
+    }
+  }
+
+  removeFromConatiner(this._freeMusics);
+  removeFromConatiner(this._freeSounds);
+  removeFromConatiner(this._musics);
+  removeFromConatiner(this._sounds);
+
   container[soundFile].unload();
```

Comment:
I don't see any unload method on `gdjs.HowlerSound`?

Reply:
This is not the `HowlerSound` but the `Howl`.

Reply:
Btw, because you called `clearContainer` before, it must have already unloaded and cleared the soundFile, so container[soundFile] is now undefined. So this will probably crash there? 
This means that `container[soundFile].unload();` and `delete container[soundFile];` are not needed at all :)

Reply:
The containers are cleared from the `HowlerSound` instances that refer to the `Howl` to make sure it is garbage collected and prevent to use a `HowlerSound` that uses a dead/unloaded `Howl` by accident.
The unload and delete delete the `Howl` from the `Howl` cache that is used in `createHowlerSound` (`this._loadesSounds`/`this._loadedMusics`).

Reply:
Ok I confused the "loaded" cache and the maps containing the channels/free sounds/musics.

In this case, prefer to avoid the generic "container" name: 
- name the first one the "loaded" container.
- you can keep the generic term "container" for the maps containing the channels/free sounds/musics.

Reply:
Also, consider renaming the two containers to `loadedHowlsForMusics` and ``loadedHowlsForSounds`, to make extra clear that these containers are containing `Howl`s, not `gdjs.HowlerSound` :)